[UpgradeFile] Introduce upgrade from Sylius 1.12 with PriceHistoryPlugin to Sylius 1.13 (#14900)
Sylius/Sylius: UPGRADE-1.13.md

Reviewer: I'd be for moving point 7. about PriceHistory at the top. It won't be possible to upgrade to Sylius 1.13 first, as the plugin's support is terminated with Sylius 1.12, and it must be removed first

6. Not passing `Sylius\Component\Core\Checker\ProductVariantLowestPriceDisplayCheckerInterface` 
   to `Sylius\Component\Core\Calculator\ProductVariantPriceCalculator`
   as a first argument is deprecated.

> 7. Starting from version 1.13 of Sylius, the `SyliusPriceHistoryPlugin` is included. 

Reviewer: ```suggestion
7. Starting with Sylius 1.13, the `SyliusPriceHistoryPlugin` is included. 
```
